### PR TITLE
AEA-1021 Update aea-deploy docker image and quickstart guide

### PR DESCRIPTION
**Pull request by marcofavorito**
## Proposed changes

Update aea-deploy docker image and quickstart guide

**Replace username of Docker image before merging.**

## Fixes

n/a

## Types of changes

What types of changes does your code introduce to agents-aea?
_Put an `x` in the boxes that apply_

- [ ] Bugfix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)

## Checklist

_Put an `x` in the boxes that apply._

- [ ] I have read the [CONTRIBUTING](../CONTRIBUTING.md) doc
- [ ] I am making a pull request against the `develop` branch (left side). Also you should start your branch off our `develop`.
- [ ] Lint and unit tests pass locally with my changes
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] I have checked that code coverage does not decrease.
- [ ] I have checked that the documentation about the `aea cli` tool works
- [ ] I have added necessary documentation (if appropriate)
- [ ] Any dependent changes have been merged and published in downstream modules

## Further comments

n/a

### CODE REVIEW

**Review by DavidMinarsch — APPROVED** [review]
LGTM, let's hope we get the `fetchai` version setup asap

File: docs/quickstart.md [hunk 1]
```diff
@@ -11,6 +11,19 @@ The AEA framework can be used on `Windows`, `Ubuntu/Debian` and `MacOS`.
...
 You need <a href="https://www.python.org/downloads/" target="_blank">Python 3.6</a> or higher as well as <a href="https://golang.org/dl/" target="_blank">Go 1.14.2</a> or higher installed.
 
+
+### Optional: using the Docker image.
+
+We also provide a Docker image with all the needed dependencies.
+
+To pull:
+```bash
+docker pull marcofavorito/aea-deploy:latest
```
Comment (DavidMinarsch) [inline docs/quickstart.md:21]:
```suggestion
docker pull fetchai/aea-deploy:latest
```

File: docs/quickstart.md [hunk 2]
```diff
@@ -362,6 +375,15 @@ Delete the AEA from the parent directory (`cd ..` to go to the parent directory)
 aea delete my_first_aea
 ```
 
+## Using the Docker image
+
+You can reproduce the quick-start guide by:
+```bash
+docker run -it marcofavorito/aea-deploy
```
Comment (DavidMinarsch) [inline docs/quickstart.md:382]:
```suggestion
docker run -it fetchai/aea-deploy 
```

**Review by DavidMinarsch — APPROVED** [review]
LGTM

File: docs/quickstart.md [hunk 3]
```diff
@@ -11,6 +11,19 @@ The AEA framework can be used on `Windows`, `Ubuntu/Debian` and `MacOS`.
 
 You need <a href="https://www.python.org/downloads/" target="_blank">Python 3.6</a> or higher as well as <a href="https://golang.org/dl/" target="_blank">Go 1.14.2</a> or higher installed.
 
+
+### Optional: using the Docker image.
+
+We also provide a Docker image with all the needed dependencies.
+
```
Comment (lrahmani) [inline docs/quickstart.md:18]:
```suggestion
To use the image you will first have to pull it and than run it with your
current local directory mounted as a docker volume. 
This allows you to keep your agents local while working on them from
within the docker container.

```

File: docs/quickstart.md (hunk 1 above)
Comment (lrahmani) [inline docs/quickstart.md]:
```suggestion
docker pull marcofavorito/aea-ubuntu:latest
```

File: docs/quickstart.md [hunk 4]
```diff
@@ -11,6 +11,19 @@ The AEA framework can be used on `Windows`, `Ubuntu/Debian` and `MacOS`.
...
+To pull:
+```bash
+docker pull marcofavorito/aea-deploy:latest
+```
+
+See at the bottom of the guide how to run the quickstart using the Docker image.
```
Comment (lrahmani) [inline docs/quickstart.md:24]:
```suggestion
```

File: docs/quickstart.md [hunk 5]
```diff
@@ -11,6 +11,19 @@ The AEA framework can be used on `Windows`, `Ubuntu/Debian` and `MacOS`.
...
+
+### Optional: using the Docker image.
+
+We also provide a Docker image with all the needed dependencies.
+
+To pull:
+```bash
+docker pull marcofavorito/aea-deploy:latest
+```
+
```
Comment (lrahmani) [inline docs/quickstart.md:23]: To run the image
​
- Linux and MacOs
  ```bash
  docker run -it -v $(pwd):/dev marcofavorito/aea-ubuntu:latest 
  ```
- Windows
  ```bash
  docker run -it -v %cd%:/dev marcofavorito/aea-ubuntu:latest 
  ```
Once succefully logged into the docker container, 
you can follow the rest of the guide the same way as if not using docker.
​

File: docs/quickstart.md [hunk 6]
```diff
@@ -362,6 +375,15 @@ Delete the AEA from the parent directory (`cd ..` to go to the parent directory)
...
 
+## Using the Docker image
+
+You can reproduce the quick-start guide by:
+```bash
+docker run -it marcofavorito/aea-deploy 
+``` 
+
+To stop, use CTRL+C.
+
```
Comment (lrahmani) [inline docs/quickstart.md]:
```suggestion
```

**Review by lrahmani — COMMENTED** [review]
suggestions are for reference, writing can be improved.

**Comment by DavidMinarsch** [comment]
Closed in favour of #1894

**Pull request closed without merging**